scripts/create-thief-resources.sh

```diff
@@ -22,7 +22,7 @@ export THIEF_KEYVAULT_NAME=${THIEF_PREFIX}-thief-kv
 ##############################
 # add the app insights az extension
 ##############################
-echo az extension add --name application-insights
+az extension add --name application-insights
```

Comment:
Consider making an ARM template instead so it works across languages/platforms. Also, in my experience the CLI breaks you fairly often.

Also try using Bicep. See my team's 1N under Testing | Bicep for some instructions.

Reply:
absolutely.  I just haven't had time to do this yet.